pawar-2018/pawar2018
# Added Proper Google Optimize Implementation

# What does this pull request do?
This pull requests adds the Google Optimize code snippet to every page of the Pawar 2018 theme, as well as adds Google Optimize's page hiding snippet and finally, initializes the Universal Analytics tracker so that we can attach our Google Optimize functionality to the tracker before sending a page view.
# How should the reviewer test this pull request?
Look in the template-parts/header-head file to ensure that Google Optimize, Google Optimize Page Hiding Snippet, Google Analytics (without a pageview send) and Google Tag Manager are all included in the file.
# Include the Trello card for this PR, if there is one.
https://trello.com/c/ii6fwXei/33-enhance-google-optimizer-to-remove-page-flicker
# Include any screenshots that will help explain this PR.
N/A
# Any another context you want to provide?
N/A
